src/around.js

```diff
@@ -0,0 +1,30 @@
+var curryN = require('./curryN');
+/**
+ * Accepts three functions, `before`, `after` and `fn`, and a single value.
+ * `R.around` effectively wraps `fn` with `before` and `after` by first passing the
+ * given value to `before`, which returns a value that is then passed to `fn`,
+ * whose return value is finally passed to `after`. The return value of
+ * `R.around` is the result of this call to `after`.
+ *
+ * This can be useful for converting some type `a` to be compatible with some
+ * function `(b -> b)` and back again, where an isomorphism exists between
+ * `a` and `b`: `R.around(aToB, bToA, bFn)`.
+ *
+ * @func
+ * @memberOf R
+ * @category Function
+ * @sig (a -> b) -> (c -> d) -> (b -> c) -> a -> d
+ * @param {Function} before
+ * @param {Function} after
+ * @param {Function} fn
+ * @param {*}        x
+ * @return {*}
+ * @example
+ *
+ *      var headToUpper = R.adjust(R.toUpper, 0);
+ *      var upperKeys = R.around(R.toPairs, R.fromPairs, R.map(headToUpper));
+ *      upperKeys({ foo: 0, bar: 1, baz: 2 }); //=> { FOO: 0, BAR: 1, BAZ: 2 }
```

Comment:
I'd recommend we change the values in this example from numbers to some strings just to demonstrate that only the keys are being upper-cased. With ints it's not clear since `'1'.toUpper() === '1'`

Naturally `.toUpper` of an int is an error, but using strings of lowercase letters as the values would definitely help me understand.